ibm_cloud_sdk_core/api_exception.py

```diff
@@ -54,7 +54,7 @@ def code(self):
         """The old `code` property with a deprecation warning."""
 
         warnings.warn(
-            'Using the `code` attribute on the `ApiException` is deprecated and'
+            'Using the `code` attribute on the `ApiException` is deprecated and '
```

Comment:
Is a space added automatically between string breaks in the `warn` method?

Reply:
They are not.  I saw this warning when running the tests and it appeared as `andwill`